install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -111,6 +111,10 @@ spec:
 {{- if .Values.global.ci.kubeCacheMutationDetector }}
         - name: KUBE_CACHE_MUTATION_DETECTOR
           value: "true"
+{{- end }}
+{{- if .Values.policyEnforcementMode }}
+        - name: CILIUM_ENABLE_POLICY
+          value: "{{ .Values.policyEnforcementMode }}"
```

Comment:
Is there a reason not to add this into the configmap? Most options are configured there instead.

Reply:
Yes, this should be in the configmap.